tracing-opentelemetry/src/lib.rs

```diff
@@ -22,18 +22,15 @@
 //! * `otel.name`: Override the span name sent to OpenTelemetry exporters.
 //! Setting this field is useful if you want to display non-static information
 //! in your span name.
-//! * `otel.kind`: Set the span kind to one of the supported OpenTelemetry
-//! [span kinds]. The value should be a string of any of the supported values:
-//! `SERVER`, `CLIENT`, `PRODUCER`, `CONSUMER` or `INTERNAL`. Other values are
-//! silently ignored.
+//! * `otel.kind`: Set the span kind to one of the supported OpenTelemetry [span kinds].
 //!
-//! [span kinds]: https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/trace/api.md#spankind
+//! [span kinds]: https://docs.rs/opentelemetry/latest/opentelemetry/trace/enum.SpanKind.html
 //!
 //! ### Semantic Conventions
 //!
 //! OpenTelemetry defines conventional names for attributes of common
 //! operations. These names can be assigned directly as fields, e.g.
-//! `trace_span!("request", "otel.kind" = "client", "http.url" = ..)`, and they
+//! `trace_span!("request", "otel.kind" = %SpanKind::Client, "http.url" = ..)`, and they
```

Comment:
nit/TIOLI: it might be worth changing this example so that it's formatted as Rust code? but we can do that later.